docs/api/webxrHardwareExtensionAPIs.md

```diff
@@ -1,16 +1,42 @@
----
+.---
 title: WebXR Hardware Extension APIs
 type: api
 layout: docs
 order: 2
 parent_section: api
 ---
 
-This currently consists of seperate device-specific APIs, but will be folded into a more generic API, properly coded in Exokit generically on the `session` objects rather than under a `magicleap` namespace.
+The WebXR Hardware Extension APIs is a generic API that is not tied to any one hardware platform and provides access to: meshing, planes detection, hand tracking, eye tracking. This is inspired by the [WebExtension API style](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API). Although it is not a web standard yet, these are finding their way into the WebXR spec and Exokit wants to be standards-compliant.
```

Comment:
It's probably not true that we're following `WebExtension API style` anymore. We're more following WebXR style.

Is there an argument for otherwise or was this just copy-paste?

Reply:
I thought that:
```js
extensions: {
    meshing: true,
    planesTracking: true,
    handTracking: true,
    eyeTracking: true,
  },
```

formatting was from the WebExtension style.

Reply:
WebExtension style is when you dangle objects off of `browser`. We used to do this for the Magic Leap case, but that's no longer the case in this PR.

`extensions` here has a totally different meaning -- in the GL extension sense. It's not WebExtension style, but "WebXR session init style".

Sorry about the confusion. Naming is the hardest problem in computer science and all that...

Reply:
Removed WebExtension style mention in 6878c36